modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -143,6 +144,7 @@ public String getDescription() {
     protected boolean skipDefaultInterface = false;
     protected boolean useTags = false;
     protected boolean useBeanValidation = true;
+    protected String customValidationAnnotationsPackage = "";
```

Comment:
are `List<String>` is better option here? then you can import custom annotations from multiple packages

Reply:
reimplemented, using ; separated list of strings, see SpringCodegenTest for update

Reply:
@borsch  actually I made changes, that You requested =)